content-repo/extra-docs/articles/setup-google-maps.md

```diff
@@ -0,0 +1,48 @@
+---
+id: google-maps
+title: Set up Google Maps in Cortex XSOAR to use map automations
+description: How to set up Google Maps in Cortex XSOAR to use map automations.
+---
+
+You need to set up Google Maps in Cortex XSOAR before using either the `ShowLocationOnMap` or the `ShowLocationOnMap` automation. To use Google Maps, you need to create a Google Cloud Plaform project that has [billing enabled](https://developers.google.com/maps/documentation/javascript/cloud-setup#billing). After creating the project, you need to enable the Maps JavaScript API and then create an API key for the project. You then need to add the API key to Cortex XSOAR. If using the `ShowLocationOnMap` automation, to view the map, you also need to add the automation to a indicator layout.  
```

Comment:
Same automation name `ShowLocationOnMap` repeated in first sentence. One should be `ShowOnMap`.